lighthouse-plugin-publisher-ads/audits/ad-render-blocking-resources.js

```diff
@@ -30,7 +30,7 @@ const UIStrings = {
   'https://developers.google.com/web/tools/lighthouse/audits/blocking-resources' +
   ').',
   failureDisplayValue: 'Up to {timeInMs, number, seconds} s tag load time ' +
-  'improvement',
+  'speed-up',
```

Comment:
Are we specifically trying to avoid the word "improvement"? If not, I think it reads better than this. If so, what's the concern?